test/CSAccounting.t.sol

```diff
@@ -2870,6 +2870,184 @@ contract CSAccountingclaimRewardsUnstETHTest is
     }
 }
 
+contract CSAccountingClaimableBondTest is CSAccountingRewardsBaseTest {
+    function test_default() public override assertInvariants {
+        _operator({ ongoing: 16, withdrawn: 0 });
+        _deposit({ bond: 32 ether });
+        _rewards({ fee: 0.1 ether });
+
+        uint256 claimableBondShares = accounting.getClaimableBondShares(0);
+
+        assertEq(
+            claimableBondShares,
+            0,
+            "claimable bond shares should be zero"
+        );
+    }
+
+    function test_WithCurve() public override assertInvariants {
+        _operator({ ongoing: 16, withdrawn: 0 });
+        _deposit({ bond: 32 ether });
+        _rewards({ fee: 0.1 ether });
+        _curve(defaultCurve);
+
+        uint256 claimableBondShares = accounting.getClaimableBondShares(0);
+
+        assertApproxEqAbs(
+            claimableBondShares,
+            stETH.getSharesByPooledEth(15 ether),
+            1 wei,
+            "claimable bond shares should be equal to the curve discount"
+        );
+    }
+
+    function test_WithLocked() public override assertInvariants {
+        _operator({ ongoing: 16, withdrawn: 0 });
+        _deposit({ bond: 32 ether });
+        _rewards({ fee: 0.1 ether });
+        _lock({ id: 0, amount: 1 ether });
+
+        uint256 claimableBondShares = accounting.getClaimableBondShares(0);
```

Comment:
Looking at the very first test I'd say there are no shares to claim even with no lock, so it's better to adjust the test, I think.

Reply:
Added extra bond. Good catch!